examples/gno.land/p/sys/validators/types.gno

```diff
@@ -27,6 +27,12 @@ type ValsetProtocol interface {
 
 	// GetValidators returns the currently active validator set
 	GetValidators() []Validator
+
+	// GetValidatorsMap returns the currently active validator set as a map
+	GetValidatorsMap() map[address]Validator
```

Comment:
This is redundant since we already have `GetValidators`, and you can reconstruct the map from it